src/qt/bitcoingui.cpp

```diff
@@ -889,16 +889,26 @@ void BitcoinGUI::createIconMenu(QMenu *pmenu)
     // Configuration of the tray icon (or dock icon) icon menu
     pmenu->addAction(toggleHideAction);
     pmenu->addSeparator();
-    pmenu->addAction(sendCoinsMenuAction);
-    pmenu->addAction(coinJoinCoinsMenuAction);
-    pmenu->addAction(receiveCoinsMenuAction);
-    pmenu->addSeparator();
-    pmenu->addAction(signMessageAction);
-    pmenu->addAction(verifyMessageAction);
+
+    if (enableWallet)
+    {
+        pmenu->addAction(sendCoinsMenuAction);
+        pmenu->addAction(coinJoinCoinsMenuAction);
+        pmenu->addAction(receiveCoinsMenuAction);
+        pmenu->addSeparator();
+        pmenu->addAction(signMessageAction);
+        pmenu->addAction(verifyMessageAction);
+    }
+
```

Comment:
Fix formatting

Reply:
I hope I did it right this time :)

Reply:
Looks better :) Added two suggestions. Need to drop the unneeded new lines

Reply:
Sounds good! I'll be more mindful of new lines in the future.